g4f/gui/client/html/index.html

```diff
@@ -36,7 +36,7 @@
 
         #message-input {
             margin-right: 30px;
-            height: 82px;
+            height: 85px;     /*  Old_code height: 82 */
```

Comment:
Please remove the comment /* Old_code height: 82px */ as well for the same reason.